src/components/cylc/gscan/GScan.vue

```diff
@@ -317,19 +311,25 @@ export default {
     sortedWorkflows () {
       return [...this.filteredWorkflows].sort((left, right) => {
         if (left.status !== right.status) {
-          if (left.status === WorkflowState.STOPPED.name) {
-            return 1
+          if (left.status === WorkflowState.RUNNING.name) {
+            return -1
           }
-          if (right.status === WorkflowState.STOPPED.name) {
+          if (left.status === WorkflowState.HELD.name && right.status !== WorkflowState.RUNNING.name) {
             return -1
           }
+          return 1
```

Comment:
There is a bug in how sorting is being done. With this I think we should get the right behaviour. Tested with two workflows, altering their states. It should be OK now :crossed_fingers: 

Will need to add more e2e/unit tests for the sorting of GScan I think. Will create an issue for that.